### DISCUSSION TITLE
Inverse Gaussian absorption

### DISCUSSION THREAD

**Discussion by rosamartae** [metrumresearchgroup/mrgsolve#1022] [Q&A]
Dear All,

I am relatively new to PK modeling and now I am trying to test the inverse gaussian absorption. My problem is that when I do multiple dose simulations, I notice an accumulation in the dosing compartment and as a result, the PK becomes non-linear in time (single dose AUCinf is not equal to multiple dose AUCtau). I am sure this is not a normal behavior, so my model must be wrong. Is this not how you implement the inverse gaussian absorption?

Thank you,
Marta

[CMT] [@annotated](url)
DEPOT  : Dosing compartment (gut)
CENT   : Central compartment
PERI   : Peripheral compartment

[PARAM]
TVVc = 700000,	
TVK12 = 0.1,   
TVK21 = 0.2,   
TVK10 = 0.03  	

TVCV = 0.8, 	// skewness
TVMAT = 4    	// mean absorption time
 
[OMEGA] @annotated 
nVc   : 0.03	: ETA on Vc
nK10  : 0.05	: ETA on K10

[SIGMA]
0

[MAIN]
double Vc = TVVc * exp(nVc);
double K12 = TVK12;
double K21 = TVK21;
double K10  = TVK10 * exp(nK10);
double CV = TVCV;
double MAT  = TVMAT;

double TAD = self.tad();
double Inp = 0;

if(TAD == 0) {
  Inp = 0;
} else {
  Inp = sqrt(MAT /(2 * 3.14 * pow(CV, 2) * pow(TAD, 3))) * exp( -1 * (pow(TAD - MAT, 2)/(2 * pow(CV,2) * MAT * TAD)));
}

double Input = Inp;

[ODE]
dxdt_DEPOT = - (DEPOT * Input);
dxdt_CENT  = (DEPOT * Input) - (CENT * K12 - PERI * K21) - (CENT * K10);
dxdt_PERI  = (CENT * K12)-(PERI * K21);

[TABLE]
capture IPRED = CENT/Vc;

[CAPTURE]
Input

![gauss](https://user-images.githubusercontent.com/94457626/194509505-d27a32f7-d217-4a6b-8149-16526c433d87.PNG)

**Comment by curtisKJ** [comment]
Did you ever get this to work? Was looking to try this recently myself, I could review your code above if no solution found. Thanks.
